localbuild.xml

```diff
@@ -0,0 +1,889 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest>
+
+  <remote  name="aosp"
+           fetch="https://android.googlesource.com/"
+           review="https://android-review.googlesource.com/" />
+  <default revision="refs/tags/android-11.0.0_r43"
+           remote="aosp"
+           sync-j="4" />
+  <remote name="sony" fetch="https://github.com/sonyxperiadev/" />
+  <remote name="NXP" fetch="https://github.com/NXP/" />
+  <remote name="exfatprogs" fetch="https://github.com/exfatprogs/" />
+  <remote name="hybris" fetch="https://github.com/mer-hybris"/>
+
+  <superproject name="platform/superproject" remote="aosp"/>
+  <contactinfo bugurl="go/repo-bug" />
+
+<!-- devices -->
+
+  <project path="device/sony/sepolicy" name="device-sony-sepolicy" groups="device" remote="sony" revision="r-mr1" />
```

Comment:
The format here is not what `repo manifest -o localbuild.xml` would output under HABUILD_SDK (`tagged-localbuilld.xml` is correct however) . I'll provide fixes later, after testing locally.

Reply:
Have you updated to `android-11.0.0_r46` as well?:)